isis/tests/FunctionalTestsLeisa2isis.cpp

```diff
@@ -63,6 +63,7 @@ TEST(leisa2isisTest, leisa2isisTestDefault) {
 
   // BandBin Group
   // Check size, first, 2 middle, and last values? Enough?
+  PvlGroup &bandbin = isisLabel->findGroup("BandBin", Pvl::Traverse);
```

Comment:
I must have accidentally deleted this when doing comment-removal cleanup. Thank you for catching and fixing this!